sphinx/domains/c/__init__.py

```diff
@@ -779,6 +797,51 @@ def get_objects(self) -> Iterator[tuple[str, str, str, str, str, int]]:
             newestId = symbol.declaration.get_newest_id()
             yield (name, dispname, objectType, docname, newestId, 1)
 
+    def intersphinx_add_entries_v2(self, store: dict,
```

Comment:
```suggestion
    def intersphinx_add_entries_v2(self, store: dict[str, Any],
```

I now understand why you had an Any for `store` because of Liskov's principle.